src/client/Pages/CommonComponents/Camera/CameraView.tsx

```diff
@@ -38,23 +39,33 @@ const CameraView = () => {
     }
   };
 
+  // TODO: Need to make sure to clean up on unmount
   useEffect(() => {
     if (!cameraStarted) {
       startCamera();
       setCameraStarted(true);
     }
-  });
+  }, [cameraStarted, startCamera]);
```

Comment:
This is probably not the correct fix since startCamera is not a state. Also cameraStarted will not be updated based on our current code.
I'm fixing the camera issue so you don't need to worry about that.